content/articles/pushserver.rst

```diff
@@ -0,0 +1,184 @@
+:title: pushserver
+:date: 2015-02-09 13:01
+:summary: Settings configuration system based on entry points of setuptools
```

Comment:
oops, sorry
done

On 20 February 2015 at 15:54, Peter Odding <notifications@github.com> wrote:

> In content/articles/pushserver.rst
> <https://github.com/paylogic/developer.paylogic.com/pull/16#discussion_r25074099>
> :
>
> > @@ -0,0 +1,184 @@
> > +:title: pushserver
> > +:date: 2015-02-09 13:01
> > +:summary: Settings configuration system based on entry points of setuptools
>
> @bubenkoff <https://github.com/bubenkoff>: Looks like this was
> copy/pasted from another article (should be obvious which one ;-). Can you
> please add a proper summary here? The archive of articles on the live site
> <http://developer.paylogic.com/archives.html> is confusing right now (the
> summary texts are shown there) because this pull request was already merged
> and is now live.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/paylogic/developer.paylogic.com/pull/16/files#r25074099>
> .
>



-- 
Anatoly Bubenkov